routing/datasource.go

```diff
@@ -389,12 +394,12 @@ func receiveRouteMatcher(o Options, out chan<- *matcher, quit <-chan struct{}) {
 			for _, err := range errs {
 				o.Log.Error(err)
 			}
-
-			mout = m
+			// TODO: What about the errors? Should we exclude the routes which has errors?
```

Comment:
How about changing `processRouteDefs` so that it will return a slice of `*Route` (i.e. no error) and a slice of `*eskip.Route` which contains the faulty ones?